package.json

```diff
@@ -61,13 +61,13 @@
     "test:shrinkwrap": "ts-node --transpile-only automation/run.ts test-shrinkwrap",
     "test:source": "cross-env BALENA_CLI_TEST_TYPE=source mocha",
     "test:standalone": "npm run build:standalone && npm run test:standalone:fast",
-    "test:standalone:fast": "cross-env BALENA_CLI_TEST_TYPE=standalone mocha",
+    "test:standalone:fast": "cross-env BALENA_CLI_TEST_TYPE=standalone mocha --config .mocharc-standalone.js",
```

Comment:
The objective of this commit is to avoid running a sizeable fraction of the test cases twice, redundantly. The CLI still intentionally runs many tests cases twice, first using the Node.js interpreter available in the PATH, and then using the built `pkg` binary (standalone zip package), in order to test the `pkg` binary too. However, this only makes sense for the end-to-end test cases that invoke CLI commands, using the [runCommand](https://github.com/balena-io/balena-cli/blob/v12.27.3/tests/helpers.ts#L183) function in `tests/helpers.ts`. The `runCommand` function decides whether to execute the built `pkg` binary as a child process, or whether to call the CLI entry point in the same mocha process, depending on the value of the BALENA_CLI_TEST_TYPE env var. Other test cases, typically unit tests, don't call the `runCommand` function, and always run in the mocha process, regardless of the value of BALENA_CLI_TEST_TYPE. It is these other test cases that were being run twice, unnecessarily. This commit avoids running these other tests cases for the `test:standalone[:fast]` script.

Before I came up with the solution of adding the `.mocharc.js` and `.mocharc-standalone.js` files, I tried a smaller change:

```
"test:standalone:fast": "cross-env BALENA_CLI_TEST_TYPE=standalone mocha tests/commands/**/*.ts",
```

... i.e., I tried specifying the desired set of test cases on the mocha command line itself. It didn't work: it would always run more tests than specified on the command line. I tried enclosing the pattern with single quotes or double quotes: no difference. I tried different patterns for different folders, I tried explicitly specifying files instead of using wildcard characters, but it would either not work for other reasons, or it would not run the test cases I wanted. Then I tried upgrading mocha from v6 to v8: no difference. (I decided the keep the mocha upgrade anyway, because it seems like a good thing.) Then I finally tested with separate configuration files and the `--config` option, and it worked fine!